plugin/trino-openlineage/src/main/java/io/trino/plugin/openlineage/OpenLineageListenerConfig.java

```diff
@@ -25,10 +25,14 @@
 
 public class OpenLineageListenerConfig
 {
+    private static final String DEFAULT_JOB_NAME_FORMAT = "${queryId}";
+    private static final Set<String> JOB_NAME_SUBSTITUTIONS = ImmutableSet.of("${queryId}", "${user}", "${source}", "${principal}");
+
     private OpenLineageTransport transport = OpenLineageTransport.CONSOLE;
     private URI trinoURI;
     private Set<OpenLineageTrinoFacet> disabledFacets = ImmutableSet.of();
     private Optional<String> namespace = Optional.empty();
+    private Optional<String> jobNameFormat = Optional.empty();
```

Comment:
Instead of creating `Optional` here, can we directly use `DEFAULT_JOB_NAME_FORMAT` as a default value.

Reply:
I thought about case like:
```java
openLineageListenerConfig.setJobNameFormat(null)
```